pkg/scheduler/core/generic_scheduler_test.go

```diff
@@ -1308,8 +1309,19 @@ func TestZeroRequest(t *testing.T) {
 
 			pluginRegistrations := []st.RegisterPluginFunc{
 				st.RegisterQueueSortPlugin(queuesort.Name, queuesort.New),
-				st.RegisterScorePlugin(noderesources.LeastAllocatedName, noderesources.NewLeastAllocated, 1),
-				st.RegisterScorePlugin(noderesources.BalancedAllocationName, noderesources.NewBalancedAllocation, 1),
+				st.RegisterScorePlugin(noderesources.LeastAllocatedName,
+					func(plArgs apiruntime.Object, fh framework.Handle) (framework.Plugin, error) {
```

Comment:
this is a test, do we actually require the options?

Reply:
I would test with options in the plugin unit tests only